### DISCUSSION TITLE
HELP changed VOD embedding feature in AMS console

### DISCUSSION THREAD

**Discussion by nhatvuong2504** [ant-media/Ant-Media-Server#5538] [Q&A] [answered]
I am using AMS community edition.
I want to change the embed feature instead of having an iframe available, just need a link to the VOD file.
Default command:
`<iframe width="560" height="315" src="https://xxx.com:5443/LiveApp/play.html?id=streams/video-2023-09-07_10-44-42.410.mp4&playOrder=vod " frameborder="0" allowfullscreen></iframe>`

Desired command:
`https://xxx.com:5443/LiveApp/play.html?id=streams/video-2023-09-07_10-44-42.410.mp4&playOrder=vod`

I found the command code in the file: /usr/local/antmedia/webapps/root/main.bb71cde338cb0c1163a0.js
I changed the Iframe function and restarted AMS but the result is still the same. the Embed code is still the IFrame

Default CODE:
`$.notify({message:o.a.getLocaleInterface().embed_code_copied_to_clipboard},{type:"success",delay:400,timer:500,placement:{from:"top",align:"right"}})},t .prototype.copyVoDEmbedCode=function(t,e,n,i){-1==i.lastIndexOf(".")&&(i+="/"+t),this.clipBoardService.copyFromContent('<iframe width= "560" height="315" src="+r.b+this.appName+"/play.html?id="+i+'&playOrder=vod" frameborder="0" allowfullscreen></iframe>'),$ .notify({message:o.a.getLocaleInterface().embed_code_copied_to_clipboard},{type:"success",delay:400,timer:500,placement:{from:"top",align:"right"}})},t. prototype.getRtmpUrl=function`

New CODE:
`$.notify({message:o.a.getLocaleInterface().embed_code_copied_to_clipboard},{type:"success",delay:400,timer:500,placement:{from:"top",align:"right"}})},t .prototype.copyVoDEmbedCode=function(t,e,n,i){-1==i.lastIndexOf(".")&&(i+="/"+t),this.clipBoardService.copyFromContent('+r.b +this.appName+"/play.html?id="+i+'&playOrder=vod'),$.notify({message:o.a.getLocaleInterface().embed_code_copied_to_clipboard},{type:"success",delay:400,timer: 500,placement:{from:"top",align:"right"}})},t.prototype.getRtmpUrl=function(t){return this.restService.getRtmpUrl`

**Comment by tremzo** [comment, marked as answer]
If you just want a link to the MP4 file, then you can use this format:

`https://xxx.com:5443/LiveApp/streams/video-2023-09-07_10-44-42.410.mp4`

All the MP4 files are in the streams folder of the application. There is a direct link to the MP4 and M3U8 files. 

The `play.html` file generates a player that loads the video file. 

Hope this helps.
